modules/python/CMakeLists.txt

```diff
@@ -33,6 +33,10 @@ set(PYTHON_SOURCES
     python-parser.h
     python-logmsg.h
     python-logmsg.c
+    python-logtemplate.h
+    python-logtemplate.c
+    python-logtemplateoptions.h
```

Comment:
Optional: It would be easier to read IMHO if it would be python-logtemplate-options.